cdap-docs/admin-manual/source/installation/cloudera/installation.rst

```diff
@@ -61,7 +61,7 @@ Run the Cloudera Manager Admin Console *Add Service* Wizard and select *CDAP*.
 When completing the Wizard, these notes may help:
 
 - *Add Service* Wizard, Page 2: **Optional Hive dependency** is for the optional CDAP
```

Comment:
"optional", just means that the component can be disabled, not that it is off by default, right?